### ISSUE TITLE
Accessibility: Empty anchor tags

### ISSUE THREAD

**Issue by aduth** [GSA/centers-of-excellence#360]
Some pages include [anchor (`<a>`) tags](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a) for which there is no content. This can cause confusion for a person using a screen reader device, or for keyboard usage, since the element can receive focus, but a name associated with the link cannot be determined.

**Steps to Reproduce:**

1. Navigate to https://coe.gsa.gov/2018/05/21/da-update-1.html
2. <kbd>Tab</kbd> through page content until reaching the "Connect" navigation link in the header
3. <kbd>Tab</kbd> once more

_Expected:_ The next focused element should be visible and named.
_Actual:_ The focused element is a non-visible, unnamed link tag.

**Affected pages:**

- [ ] https://coe.gsa.gov/2018/05/21/da-update-1.html
- [ ] https://coe.gsa.gov/2018/05/18/cc-update-1.html

...[many others where anchor tags are constructed without an appropriate closing tag](https://github.com/GSA/centers-of-excellence/search?q=target%3D%22_blank%22+rel%3D%22noopener+noreferrer%22&unscoped_q=target%3D%22_blank%22+rel%3D%22noopener+noreferrer%22). In some cases, the closing tag appears that it may be added automatically, possibly by the static site generator tool in converting the markdown files.

**WCAG Success Criteria (A):** [4.1.2: Name, Role, Value](https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html)

**Relevant technique:** [HTML H91: Using HTML form controls and links](https://www.w3.org/WAI/WCAG21/Techniques/html/H91.html)

**Remediation Guidance:**

Content or an equivalent name should be provided for the link, and it should be visibly present in the page. It should be removed if it is not intended to be present, or it should be made to wrap content which it is intended to link.

From the code, it appears it may be intended that the link should be wrapping the image. Note that based on usage, this could also be related to #358. Currently, there is no closing tag for the link.

https://github.com/GSA/centers-of-excellence/blob/45862c0cc20e50b788c04ce8ba99f5412de7ce4f/_posts/2018-05-21-da-update-1.md#L7-L8
